.travis.yml

```diff
@@ -1,7 +1,5 @@
-# Based on http://conda.pydata.org/docs/travis.html and https://github.com/travis-ci/travis-ci/issues/9069#issuecomment-425720905
-language: python
-sudo: required
-dist: xenial
+language: minimal
+sudo: false
```

Comment:
I couldn't manage to get 3.7 working with `sudo: false` (but admittedly didn't try for that long)